client/src/components/main/Home.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { push } from 'react-router-redux';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+/** ============================================================
+ * Define Store Modules
+ * =============================================================
+ */
+import {
+  increment,
+  incrementAsync,
+  decrement,
+  decrementAsync,
+  selectHandler,
+  inputHandler
+} from '../../store/modules/counter';
+
+import { Grid, Row, Col } from 'react-bootstrap';
+import ExploreMap from './maps/ExploreMap.jsx';
+import PostList from './PostList.jsx';
+
+//Define 
+const NO_SELECT_MSG = 'No Selection';
+const NO_INPUT_MSG = 'No Message';
+
+const Home = (props) => {
+  let playGround = false;
+
+  if (playGround) {
```

Comment:
This looks like code that was copy and pasted from a tutorial or another app. Please remove everything that is not relevant specifically to your end product.